release-notes/Cloud/DCP_change_log.md

```diff
@@ -9,6 +9,10 @@ The dataminer.services platform gets updated continuously. This change log can h
 > [!NOTE]
 > Many features on dataminer.services are dependent on DxMs. You can find the change logs for these under [DxM release notes](xref:DxM_RNs_index).
 
+#### 10 June 2024 - Fix - ToC checkbox - Fixed an issue with the ToC URL [ID_39895]
+
+The link of our ToC was updated to the correct link.
```

Comment:
Could you rephrase this so it's clear what the issue was? As a user, I wouldn't have a clue what is meant with the current sentence.

Reply:
Discussed in chat, updated it with the given suggestion